src/ukify/ukify.py

```diff
@@ -704,6 +706,34 @@ def make_uki(opts):
     print(f"Wrote {'signed' if sign_args_present else 'unsigned'} {opts.output}")
 
 
+UKI_DEFAULT_SECTIONS = {
+    'binary': set(['.linux', '.initrd', '.splash', '.dtb']),
+    'text' : set(['.osrel', '.uname', '.cmdline', '.pcrpkey', '.pcrsig']),
+}
+
+
+def pe_read_sections(uki: str, output: pathlib.Path):
+    pe = pefile.PE(uki, fast_load=True)
+    out = f"Sections in {uki}:\n"
+    for section in pe.sections:
+        name = section.Name.decode().rstrip('\x00')
+        if name in UKI_DEFAULT_SECTIONS['binary']:
+            out += f"{name}: present\n"
+        elif name in UKI_DEFAULT_SECTIONS['text']:
+            start = section.PointerToRawData
+            end = start + section.Misc_VirtualSize
+            data = pe.__data__[start:end].decode()
+            out += f"{name}:\n{data}\n"
+    if output:
+        output.open("w").write(out)
+    else:
+        print(out, end='')
```

Comment:
```suggestion
    if output:
        output.write_text(out)
    else:
        print(out, end='')
```
Also the typing is off, from the function signature `output` is always a `pathlib.Path` and can't be falsey.